server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/redirect/NotifyMailetsMessage.java

```diff
@@ -88,12 +95,23 @@ public String generateMessage(String parameterMessage, Mail originalMail) throws
         return builder.toString();
     }
 
+    private String safeDecode(String text) {
+        try {
+            return MimeUtility.decodeText(text);
+        } catch (UnsupportedEncodingException e) {
+            LOGGER.debug("Could not decode following header value {}", text, e);
```

Comment:
header value ?

Reply:
and why debug ?

Reply:
What level do you think you should use WARM ERROR ?
Yes this method is only meant to decode header value otherwise no ?

Reply:
I use error and just speak about "value" instead of "header value"